server/server.py

```diff
@@ -1,21 +1,38 @@
 from flask import Flask, request, jsonify
 from newsapi import NewsApiClient
 
+import os
+from dotenv import load_dotenv
+
+# Load environment variables from .env file
+load_dotenv()
+
 #Init Flask App
 app = Flask(__name__)
 
 #Init News Api
-newsapi = NewsApiClient(api_key = '31673bb1bed245e8b253b67389155bb2')
-
-
+newsapi = NewsApiClient(api_key=os.getenv("NEWS_API_KEY"))
 
+#API endpoint for handling search request based on user input
 @app.route("/search", methods = ["GET", "POST"])
```

Comment:
```suggestion
@app.route("/search", methods = ["GET"])
```
bỏ POST vì mình chưa dùng đến